bears/python/PEP8Bear.py

```diff
@@ -49,6 +49,6 @@ def run(self, filename, file,
 
         for diff in diffs:
             yield Result(self,
-                         "The code does not comply to PEP8.",
+                         "The code does not comply to PyCodeStyle.",
```

Comment:
well but you don't comply to a tool, you comply to some guidelines don't you? I'm not content with this yet...